python_modules/libraries/dagster-powerbi/dagster_powerbi/translator.py

```diff
@@ -281,7 +281,7 @@ def get_semantic_model_spec(self, data: PowerBITranslatorData) -> AssetSpec:
         dataset_id = data.properties["id"]
         source_ids = data.properties.get("sources", [])
         source_keys = [
-            self.get_data_source_spec(
+            self.get_asset_spec(
```

Comment:
this is an unrelated (but also related) bugfix, if i'm understanding correctly?

Reply:
correct